src/main/java/at/hannibal2/skyhanni/api/GetFromSackAPI.kt

```diff
@@ -222,12 +223,25 @@ object GetFromSackAPI {
     }
 
     @SubscribeEvent
-    fun onRepoReload(event: RepositoryReloadEvent) {
-        sackListInternalNames = event.getConstant<SacksJson>("Sacks").sackItems.toSet()
+    fun onNeuRepoReload(event: NeuRepositoryReloadEvent) {
+        val data = event.getConstant("sacks") ?: ErrorManager.skyHanniError("NEU sacks data is null.")
+        try {
+            val sacksData = ConfigManager.gson.fromJson<NeuSacksJson>(data).sacks
+            val uniqueSackItems = mutableSetOf<NEUInternalName>()
+
+            sacksData.values.forEach { sackInfo ->
+                sackInfo.contents.forEach { content ->
+                    uniqueSackItems.add(content)
+                }
+            }
+
```

Comment:
Missing the set of the InternalNames set
```suggestion

            sackListInternalNames = uniqueSackItems.map { it.asString() }.toSet()
```